views/elements/course-filters.php

```diff
@@ -0,0 +1,196 @@
+<?php
+/**
+ * Course Filter views
+ *
+ * A common filter element for all the backend pages
+ *
+ * @package Tutor\Views
+ * @subpackage Tutor\ViewElements
+ * @author Themeum <support@themeum.com>
+ * @link https://themeum.com
+ * @since 3.5.0
+ */
+
+use TUTOR\Input;
+
+if ( isset( $data ) ) : ?>
+<div class="tutor-admin-container tutor-admin-container-lg">
+	<div class="tutor-wp-dashboard-course-filter tutor-justify-<?php echo esc_attr( isset( $data['bulk_action'] ) && true === $data['bulk_action'] ? 'between' : 'end' ); ?>">
+		<?php if ( isset( $data['bulk_action'] ) && true === $data['bulk_action'] ) : ?>
+		<form id="tutor-admin-bulk-action-form" action method="post">
+			<input type="hidden" name="action" value="<?php echo esc_html( $data['ajax_action'] ); ?>" />
+			<div class="tutor-d-flex">
+				<div class="tutor-mr-12">
+					<select name="bulk-action" title="Please select an action" class="tutor-form-control tutor-form-select">
+						<?php foreach ( $data['bulk_actions'] as $k => $v ) : ?>
+							<option value="<?php echo esc_attr( $v['value'] ); ?>">
+								<?php echo esc_html( $v['option'] ); ?>
+							</option>
+						<?php endforeach; ?>
+					</select>
+				</div>
+				<button class="tutor-btn tutor-btn-outline-primary" id="tutor-admin-bulk-action-btn" data-tutor-modal-target="tutor-bulk-confirm-popup">
+					<?php esc_html_e( 'Apply', 'tutor' ); ?>
+				</button>
+			</div>
+		</form>
+		<?php endif; ?>
+		<?php
+		$search_query  = Input::get( 'search', '' );
```

Comment:
Use appropriate type casting for all the input fields like the following
`$search_query  = Input::get( 'search', '', Input::TYPE_STRING );`